Nosenti/Innocent-Ingabire-portfolio
# Ft crud

#### What does this PR do?
This PR completed the CRUD operation of the blog, Get Contact messages to firebase and authenticate Admin.
#### Description of Task to be completed
Read, Delete, Save and Delete Blogs
Read data sent by user via different forms from the database
Authenticate Admin to use Admin dashboard
[the tasks that you completed on the feature]
#### How should this be manually tested?
Clone the repository
Upon landing on the first page, you will be asked to confirm whether you want the side to track your location or not.
Send the data via different forms and you will get the message successful message when they are saved to the database
#### Any background context you want to provide?
Updating the profile is not complete yet.
### What are the relevant stories?
<blockquote class="trello-card"><a href="https://trello.com/c/q989oDZS/22-update-the-blog">Update the blog</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
